erpnext/projects/doctype/project/project.py

```diff
@@ -227,9 +253,10 @@ def send_welcome_email(self):
 				user.welcome_email_sent=1
 
 	def on_update(self):
-		self.load_tasks()
-		self.sync_tasks()
-		self.update_dependencies_on_duplicated_project()
+		# self.load_tasks()
+# 		self.sync_tasks()
```

Comment:
Is this purposefully commented?

Reply:
Yes added in validate and divide sync_tasks function, 
added update_costing_and_percentage_complete function which is added in on_update function